web_app/api/serializers/user.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel, Field
+
+class CheckUserResponse(BaseModel):
+    is_contract_deployed: bool = Field(..., example=False, description="Indicates if the user's contract is deployed.")
+
+
+class UpdateUserContractResponse(BaseModel):
+    is_contract_deployed: bool = Field(..., example=False, description="Indicates if the user's contract is deployed.")
+
+class GetUserContractAddressResponse(BaseModel):
```

Comment:
Please, add blank line here to keep formatting

Reply:
oh okay.